source/ProcessManager.Client.Tests/Fixtures/ProcessManagerClientFixture.cs

```diff
@@ -0,0 +1,104 @@
+﻿// Copyright 2020 Energinet DataHub A/S
+//
+// Licensed under the Apache License, Version 2.0 (the "License2");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System.Diagnostics;
+using System.Diagnostics.CodeAnalysis;
+using Azure.Identity;
+using Azure.Messaging.ServiceBus.Administration;
+using Energinet.DataHub.Core.FunctionApp.TestCommon.Azurite;
+using Energinet.DataHub.Core.FunctionApp.TestCommon.Configuration;
+using Energinet.DataHub.Core.FunctionApp.TestCommon.ServiceBus.ResourceProvider;
+using Energinet.DataHub.ProcessManager.Core.Tests.Fixtures;
+using Energinet.DataHub.ProcessManager.Orchestrations.Tests.Fixtures;
+using Energinet.DataHub.ProcessManager.Tests.Fixtures;
+using Xunit.Abstractions;
+
+namespace Energinet.DataHub.ProcessManager.Client.Tests.Fixtures;
+
+public class ProcessManagerClientFixture : IAsyncLifetime
+{
+    public ProcessManagerClientFixture()
+    {
+        var taskHubName = "ClientsTest01";
+
+        DatabaseManager = new ProcessManagerDatabaseManager("ProcessManagerClientTests");
+
+        IntegrationTestConfiguration = new IntegrationTestConfiguration();
+
+        AzuriteManager = new AzuriteManager(useOAuth: true);
+
+        OrchestrationsAppManager = new OrchestrationsAppManager(
+            DatabaseManager,
+            IntegrationTestConfiguration,
+            AzuriteManager,
+            taskHubName,
+            8101,
+            false);
```

Comment:
When we use numbers and booleans in parameters consider to show the parameter name, as there is no "context" so adding it makes it easier to review.
```suggestion
            port: 8101,
            manageAzurite: false);
```